### ISSUE TITLE
./wakeWordAgent -e sensory (command not found) HELP!!

### ISSUE THREAD

**Issue by Luke19** [alexa/alexa-avs-sample-app#363]
i have set up everything else to do with alexa perfectly on my Raspberry Pi but when i get to the last step (setting up the wake word) when using the command "cd wakeWordAgent/src && ./wakeWordAgent -e sensory" it comes back saying the command/directory can't be found?? can anyone help? 

i'm a beginner here too so please reply with something i will understand!

**Comment by Kittyjuls** [comment]
Need a bit more info,  sounds like you could be running the command from the wrong directory.  If you check your current working directory it should be ~/Desktop/alexa-avs-sample-app/samples/  Then check you cloned everything successfully by listing its contents. You should have **androidCompanionApp**  **companionService  iOSCompanionApp  javaclient  wakeWordAgent** then try the "cd wakeWordAgent/src && ./wakeWordAgent -e sensory" again.

**Comment by Luke19** [comment]
thanks for the reply
when i type the command to setup the sensory wake word i get this... (see image below)
![20161108_182422 1](https://cloud.githubusercontent.com/assets/23341598/20111694/14056792-a5e1-11e6-8c5e-db26b946b7dc.jpg)

**Comment by Luke19** [comment]
Alexa works perfectly when i click "listen" on the board but i get this ^ when i try to setup the wake word like it says in the step-by-step guide.

**Comment by Kittyjuls** [comment]
Ok, kind of looks ok, can you show what the contents of this directory? Should look something like this..
![wakeword](https://cloud.githubusercontent.com/assets/6146969/20115997/663b4d68-a5f2-11e6-9c50-a0c0eef71c06.png)

Also what happens if you use -e kitt_ai?

**Comment by Luke19** [comment]
This is what i get for the contents...
![20161108_204040 1](https://cloud.githubusercontent.com/assets/23341598/20116374/eb5d687c-a5f3-11e6-9a30-465be87608e2.jpg)

When i use the -e kitt_ai i get exactly the same problem

**Comment by Kittyjuls** [comment]
Ah, your script hasn't compiled wakeWordAgent (seen in my screen grab in green at the bottom), it might be easiest to re-run the automated_install.sh and let us know if you get any errors towards the end of the script when wakeWordAgent is processed by cmake.

**Comment by Luke19** [comment]
I re-compiled the script and it added the word agent however, now I get the note "illegal command"?? 

Alexa also doesn't work now saying "java script error" or something like that

**Comment by goodpunk6** [comment]
I also have this same issue.  re-running the automated install script did not fix the issue

**Comment by catsailor411** [comment]
Thanks goodpunk6, I'm new to the github thing and hope someone has a fix for this. I have no answer yet and am trying a few things. I did read that a guy had some luck with NOOBS, which I am trying, but that might be a dead end too. It seems odd that the same packages on the same fresh installs do not work consistently. This thing is supposed to use VLC quite a bit so I am going to verify that shortly too. Again, I am a real beginner here swinging in the dark!
My next attempt would be to try Jesse again, strip out Java 1st, then run apt-get update and apt-get upgrade twice between each step.
best of luck to you and if I find an answer I will share that with you as well. FYI, I have a model 3 pi, usb soundcard with mic and speaker port  - ear bud attached, usb micro speaker, wireless keyboard dongle, wireless mouse dongle, Sandisk 16gb sd card. I have read not to ssh into this so I'm guessing vnc'ing into this is not a good idea.

**Comment by Luke19** [comment]
anyone worked out how to fix the problem when running ./wakeWordAgent -e sensory and getting "illegal command" at the bottom??

**Comment by goodpunk6** [comment]
Sorry bud, still waiting on the error above

**Comment by Luke19** [comment]
Now I get this error...

INFO:main: Starting Wake Word Agent INFO:WakeWordAgent: State set to IDLE(2) INFO:Initializing Sensory library | library name: TrulyHandsfree | library version: 5.0.0-avs.1 | model file: ../ext/resources/spot-alexa-rpi.snsr 
Illegal instruction

**Comment by Luke19** [comment]
Anyone know why and how to fix it??

**Comment by keleric** [comment]
Closing as duplicate of #375
